x-pack/plugins/security_solution/server/cloud_posture/routes/score.ts

```diff
@@ -173,20 +176,23 @@ export const getScoreRoute = (router: SecuritySolutionPluginRouter, logger: Logg
     async (context, _, response) => {
       try {
         const esClient = context.core.elasticsearch.client.asCurrentUser;
-        const latestRunId = await getLatestRunId(esClient);
-        const allFindingsStats = await getAllFindingsStats(esClient, latestRunId);
-        const statsPerBenchmark = await getScorePerBenchmark(esClient, latestRunId);
-        const evaluationsPerFilename = await getEvaluationPerFilename(esClient, latestRunId);
-
+        const latestCycleID = await getLatestCycleId(esClient);
+        if (typeof latestCycleID === 'undefined') {
+          throw 'cycle id is missing';
+        }
+        const allFindingsStats = await getAllFindingsStats(esClient, latestCycleID);
+        const statsPerBenchmark = await getScorePerBenchmark(esClient, latestCycleID);
+        const evaluationsPerFilename = await getEvaluationPerFilename(esClient, latestCycleID);
+        const body: CloudPostureStats = {
+          totalFindings: allFindingsStats.totalFindings,
+          postureScore: allFindingsStats.postureScore,
+          totalPassed: allFindingsStats.totalPassed,
+          totalFailed: allFindingsStats.totalFailed,
```

Comment:
```suggestion
  ...allFindingsStats,
```

since all the keys in `allFindingsStats` are re-assigned in `body`, we can use [spread](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax)

Reply:
done